assets/js/aplus.js

```diff
@@ -86,217 +68,240 @@ $(function() {
-    var wrapper = $('<div></div>');
-    wrapper.append(codeBlock);
-    $(this).replaceWith(wrapper);
-
-    // Use $(element).highlightCode{noCopy: true} to prevent copy button
-    if (!options || !options.noCopy) {
-      var buttonContainer = $('<p></p>').prependTo(wrapper);
-      var copyButtonContent = $('<span class="glyphicon glyphicon-copy" aria-hidden="true"></span>');
-      var copyButtonText = $('<span></span>').text('Copy to clipboard');
-      var copyButton = $('<button data-clipboard-target="#clipboard-content-' + copyTargetCounter + '" class="btn btn-xs btn-primary" id="copy-button-' + copyTargetCounter + '"></button>');
-      copyButtonContent.appendTo(copyButton);
-      copyButtonText.appendTo(copyButton);
-      copyButton.appendTo(buttonContainer);
+    var copyTargetCounter = 0;
 
-      var hiddenTextarea = $('<textarea id="clipboard-content-' + copyTargetCounter + '" style="display: none; width: 1px; height: 1px;"></textarea>').text(codeBlock.text());
-      hiddenTextarea.appendTo(buttonContainer);
+    $.fn.highlightCode = function(options) {
 
-      // clipboard.js cannot copy from invisible elements
-      copyButton.click(function() {
-        hiddenTextarea.show();
-      });
+        return this.each(function() {
+            var codeBlock = $(this).clone();
+            var wrapper = $('<div></div>');
+            wrapper.append(codeBlock);
+            $(this).replaceWith(wrapper);
+
+            // Use $(element).highlightCode{noCopy: true} to prevent copy button
+            if (!options || !options.noCopy) {
+                var buttonContainer = $('<p></p>').prependTo(wrapper);
+                var copyButtonContent = $('<span class="glyphicon glyphicon-copy" aria-hidden="true"></span>');
+                var copyButtonText = $('<span></span>').text('Copy to clipboard');
+                var copyButton = $('<button data-clipboard-target="#clipboard-content-' + copyTargetCounter + '" class="btn btn-xs btn-primary" id="copy-button-' + copyTargetCounter + '"></button>');
+                copyButtonContent.appendTo(copyButton);
+                copyButtonText.appendTo(copyButton);
+                copyButton.appendTo(buttonContainer);
+
+                var hiddenTextarea = $('<textarea id="clipboard-content-' + copyTargetCounter + '" style="display: none; width: 1px; height: 1px;"></textarea>').text(codeBlock.text());
+                hiddenTextarea.appendTo(buttonContainer);
+
+                // clipboard.js cannot copy from invisible elements
+                copyButton.click(function() {
+                    hiddenTextarea.show();
+                });
 
-      var clipboard = new Clipboard('#copy-button-' + copyTargetCounter);
-      clipboard.on("error", function(e) {
-          hiddenTextarea.hide();
-      });
-      clipboard.on("success", function(e) {
-          hiddenTextarea.hide();
-      });
+                var clipboard = new Clipboard('#copy-button-' + copyTargetCounter);
```

Comment:
Can you also replace `var`s that are really `const`s with consts?

Reply:
At least for short and simple parts of the code (places where it's super clear when just scrolling the document.